operator/gitops/argocd/pipeline-service/tekton-results/base/kustomization.yaml

```diff
@@ -50,10 +51,20 @@ patchesJson6902:
           - "/bin/bash"
           - "-c"
           - |
-            if oc get secret -n tekton-results tekton-results-database 2>&1 >/dev/null; then exit; fi
-            oc create secret generic tekton-results-database \
+            if oc get secret -n tekton-results tekton-results-postgres 2>&1 >/dev/null; then
```

Comment:
Is there a reason why we need this init container? I am really worried that in staging/prod we are relying on various systems to land the Postgres secret at the right time. In addition, I am not comfortable with an apiserver having admin permissions over Secrets.

I'd rather the apiserver crashloop if the Postgres secret isn't there. Let our scripts/app-interface/etc. be responsible for giving us the db creds.

Reply:
This was used as a quick and dirty fix  to make sure db secret is created (for a local dev db access) before the deployment of the results-api otherwise results-api would fail to start.
Now that this code is going to be used in a staging/prod env, and we need to sync secret creation and results-api deployment in a different way and more importantly results-api should fail if the secret is not found